### PR TITLE
Reset color when the index is 9

### PR DESCRIPTION
**Pull request by aladjadj**
Match [_9m color to reset color with inherit css attribute

Fixes: #1

### CODE REVIEW

**Comment by TristanCacqueray** [comment]
I don't know yet what this 9 value means, but it does not looks like a 4bit color code.

Then using inherit does not seems to work in nested context, for example running this command: 

```ShellSession
$ echo -e "\x1b[32;1m  Green  \x1b[39;1m   inherited   \x1b[0m"
```

Display Green in green, and inherited in white. But the html encoding will be:

```html
<span style="color: green">Green<span style="color: inherit">inherited</span></span>
```

... which does not make the inherited field white.

**Comment by TristanCacqueray** [comment]
So according to the first paragraph of https://en.wikipedia.org/wiki/ANSI_escape_code#3-bit_and_4-bit , `9` is used to reset the color: `To reset colors to their defaults, use ESC[39;49m (not supported on some terminals)`

File: src/Ansi.re [hunk 1]
```diff
@@ -45,6 +45,7 @@ module AnsiCode = {
     | 05 => "magenta"->Some
     | 06 => "cyan"->Some
     | 07 => "white"->Some
+    | 09 => "inherit"->Some
```
Comment (TristanCacqueray) [inline src/Ansi.re:48]:
```suggestion
    | 09 => "initial"->Some
```

Using initial seems like the expected behavior.

**Review by TristanCacqueray — CHANGES_REQUESTED** [review]

**Comment by aladjadj** [comment]
I just want to verify if the value "initial" revert to color asign to a balise type.

Like .myclass span { color:black; } or to initial color of navigator.

in prevision of  dark theme or terminal like background

**Comment by TristanCacqueray** [comment]
Initial does not seems to use the class style, e.g.:

```html
<html>
  <head>
  <style>
.myclass {
  color: pink;
}
  </style>
  </head>
  <body>
    <span style="color: green">
      Green
      <span class=myclass style="color: initial">inherited</span>
    </span>
  </body>
</html>
```

... display `inherited` in black instead of pink

**Comment by TristanCacqueray** [comment]
Alternatively, it seems like we could use a new AnsiCode constructor to combine Clear and Style, and we could adapt the render function to terminate the previous style in https://github.com/softwarefactory-project/re-ansi/blob/master/src/Ansi.re#L257-L262 . 

Though that would be a significant change, so it would be good to look for a simpler css trick first.

**Comment by aladjadj** [comment]
I'm ok with you for "initial" value.
In the futur, it's maybe possible to add un "primary" and "secondary" colors like "black" and "white" and replace "initial" by the primary variable. It's may be more intuitive to customize style on top of app

**Review by TristanCacqueray — APPROVED** [review]

**Comment by TristanCacqueray** [comment]
Alright, then let's go with the `initial` value for now. Thank you for your contribution!

**Pull request merged**